docs/shared/config/cors.mdx

```diff
@@ -12,6 +12,6 @@ cors:
   - GET
   - POST
   - OPTIONS
-  origins:
+  policies:
```

Comment:
this doesnt seem correct now? same for most of the config changes in the docs. `policies` should have a list of policy objects, not URLs